docs/source/data_loaders.rst

```diff
@@ -0,0 +1,85 @@
+.. _dataloader-section:
+
+Acquire and Load Data
+========================
+
+UrbanAccess has data downloaders and data loaders that can be used to acquire and load transit and street network datasets:
+
+1. Transit Network: General Transit Feed Specification (GTFS)
+
+* :ref:`Download <gtfs-download>`
+* :ref:`Load <gtfs-loader>`
+
+2. Street Network: OpenStreetMap (OSM)
+
+* :ref:`Download and Load <osm-download-load>`
+
+.. _gtfs-data-object:
+
+The GTFS Data Object
+~~~~~~~~~~~~~~~~~~~~~~
+
+The GTFS data object stores the processed and aggregated GTFS feed data in Pandas dataframes.
+
+.. autodata:: urbanaccess.gtfs.gtfsfeeds_dataframe.gtfsfeeds_df
+
+.. _gtfs-download:
+
+Downloading GTFS Data
+~~~~~~~~~~~~~~~~~~~~~~
+
+Manage and download multiple feeds at once using the ``feeds`` object.
+
+.. autodata:: urbanaccess.gtfsfeeds.feeds
```

Comment:
Same as above about the `:annotation:` option.

So here, I think it might be confusing that there's documentation for both the `feeds` object and the `urbanaccess_gtfsfeeds` class. Maybe just include the class, or include both but note that the `feeds` object is just an instance of the class?